recipe/meta.yaml

```diff
@@ -28,6 +29,7 @@ requirements:
     - qcelemental <=0.28
     - qcportal >=0.54
     - openff-toolkit-base >=0.12
+    - openff-units ~0.2.1
```

Comment:
+1 good idea keeping it on 0.2.x for now, easier to expand than contract support